src/context/mutations.js

```diff
@@ -0,0 +1,13 @@
+import gql from 'graphql-tag';
+
+export const EVENTS_MUTATION = gql`
+   mutation CreateEvent($input: CreateEventInput!) {
+    createEvent(input: {tourId: 1, name: "TESTING", venueId: 3, startTime: "2021-04-10", endTime: "2021-05-01"}) {
+        tourId,
+        name,
+        venueId,
+        startTime,
+        endTime
+      }
+  }
+`
```

Comment:
Dang GraphQL, why you gotta be putting dolla signs in front of stuff, making me think this is jQuery or something?